### PR TITLE
Add Snapcraft docs

### PR DESCRIPTION
**Pull request by RoseBlume**
#### Description

- What does this PR change? Give us a brief description. 
- Closes #

### CODE REVIEW

**Comment by RoseBlume** [comment]
I intend to add nix, pacstall and ubuntu distribution documentation in the future. Should I create a linux subcategory?

**Comment by FabianLars** [comment]
> Should I create a linux subcategory?

Hmm, i'd say keep them all on the same level for now and we'll worry about it later, but @simonhyll is our sidebar king

**Comment by RoseBlume** [comment]
I can add a snap installation guide for a very large number of distributions, which ones should I limit the guide to?

**Comment by FabianLars** [comment]
i don't know exactly which install instructions you mean but what about the classic 3 debian/ubuntu, arch and fedora?

**Comment by RoseBlume** [comment]
This is now ready for review. @FabianLars

**Comment by RoseBlume** [comment]
Do not merge for now.

**Comment by RoseBlume** [comment]
I understand that there are plans to bundle snaps directly with Tauri. These docs are only meant to be temporary until the other docs can be merged.

**Comment by RoseBlume** [comment]
I am done working on this, if you can review it and tell me if anything needs to be changed, that would be appreciated

**Comment by FabianLars** [comment]
> I understand that there are plans to bundle snaps directly with Tauri. These docs are only meant to be temporary until the other docs can be merged.

At this point i almost doubt we'll see that.

File: src/content/docs/distribute/snapcraft.mdx
Comment (FabianLars) [inline src/content/docs/distribute/snapcraft.mdx]: I assume this means Tauri v1? We keep these docs v2 only so it's fine to remove this.

File: src/content/docs/distribute/snapcraft.mdx
Comment (FabianLars) [inline src/content/docs/distribute/snapcraft.mdx]: same here then. Is core24 a thing already? (and does it work for tauri v2?)

File: src/content/docs/distribute/snapcraft.mdx
Comment (FabianLars) [inline src/content/docs/distribute/snapcraft.mdx]: same (v1 mention)

File: src/content/docs/distribute/snapcraft.mdx
Comment (RoseBlume) [reply src/content/docs/distribute/snapcraft.mdx]: Ok will do, I will make separate docs for v1

File: src/content/docs/distribute/snapcraft.mdx
Comment (RoseBlume) [reply src/content/docs/distribute/snapcraft.mdx]: Core24 is a thing, and theoretical since I'm not using any plugins or extensions it should work, I can perform some testing if you want?

File: src/content/docs/distribute/snapcraft.mdx
Comment (FabianLars) [reply src/content/docs/distribute/snapcraft.mdx]: If you'd like sure, but don't worry about it too much, at this point we don't care about v1 docs _that_ much anymore tbh (we're close to v2 release-candidate)

File: src/content/docs/distribute/snapcraft.mdx
Comment (FabianLars) [reply src/content/docs/distribute/snapcraft.mdx]: If you have the time sure, if not then we can stick with 22 for now

Comment (RoseBlume) [reply src/content/docs/distribute/snapcraft.mdx]: 24 does work however snapcraft recommend 22 and I do as well.

Comment (FabianLars) [reply src/content/docs/distribute/snapcraft.mdx]: ok

**Comment by FabianLars** [comment]
btw, you can add the guide to the `.prettierignore` file if it's getting too annoying (there are a few pages in there already), the important thing is that netlify can build the docs (running `pnpm build` locally should behave the same).

**Comment by FabianLars** [comment]
And yes, i fixed the format workflow _today_ which is why you didn't have to deal with it in the other PRs yet.

**Comment by RoseBlume** [comment]
Ok I got it working. It should be good to merge now.

**Comment by RoseBlume** [comment]
Just had to fix the last mention of Tauri V1. It should be good now.

**Comment by RoseBlume** [comment]
Thanks for the help

**Review by FabianLars — APPROVED** [review]

**Comment by FabianLars** [comment]
Thanks and sorry for the delay :)

**Pull request merged**
